stan/math/prim/arr.hpp

```diff
@@ -14,6 +14,11 @@
 #include <stan/math/prim/arr/err/check_matching_sizes.hpp>
 #include <stan/math/prim/arr/err/check_nonzero_size.hpp>
 #include <stan/math/prim/arr/err/check_ordered.hpp>
+
+#include <stan/math/prim/arr/err/is_matching_size.hpp>
```

Comment:
can we alphabetize and drop the surrounding whitespace?

More concretely, 
```
#ifdef STAN_OPENCL
#include <stan/math/prim/arr/err/check_opencl.hpp>
#endif
```
should go before the `is_*` functions.